### PR TITLE
💄 Style broken iframe-example

### PR DESCRIPTION
**Pull request by dlemm**
fix broken example in iframe in examples/e-commerce/amp_for_e-commerce_getting_started and delete unnecessary p-tags in /documentation/examples/e-commerce/product_page in order to to show code-type in text

### CODE REVIEW

File: examples/source/e-commerce/Product_Browse_Page.html [hunk 1]
```diff
@@ -72,6 +72,56 @@
...
+           text-decoration: none;
+           color: #005AF0;
+         }
+
+         .ap--content a {
+           position: relative;
+           z-index: 10;
+         }
+
+         amp-selector.ampTabContainer.ampstart-headerbar-nav.i-amphtml-element.i-amphtml-layout-container.i-amphtml-layout {
```
Comment (matthiasrohmer) [inline examples/source/e-commerce/Product_Browse_Page.html]: Styling on `.i-amphtml-*` classes is forbidden.

File: examples/source/e-commerce/Product_Page.html [hunk 2]
```diff
@@ -44,6 +44,14 @@
     #product-description {
       clear: both;
     }
+
+    .ap-o-sampler--tab {
```
Comment (matthiasrohmer) [inline examples/source/e-commerce/Product_Page.html]: This selector seems strange inside the sample source - what's its purpose?

File: examples/source/e-commerce/Product_Page.html [hunk 3]
```diff
@@ -163,6 +171,55 @@
...
+      text-decoration: none;
+      color: #005AF0;
+    }
+
+    .ap--content a {
+      position: relative;
+      z-index: 10;
+    }
+
+    amp-selector.ampTabContainer.ampstart-headerbar-nav.i-amphtml-element.i-amphtml-layout-container.i-amphtml-layout {
```
Comment (matthiasrohmer) [inline examples/source/e-commerce/Product_Page.html]: Same as above.

**Comment by dlemm** [comment]
Styles have been further adapted and optimized. 

before1
before2
before3
after1
after2

**Pull request merged**